docs/book/src/introduction.md

```diff
@@ -1 +1,91 @@
+
+## Tilt-based development environment
+
+See [developer guide](https://cluster-api-ibmcloud.sigs.k8s.io/developer/tilt.html) section for details.
+
+
+## Documentation
+
+Please see our [Book](https://cluster-api-ibmcloud.sigs.k8s.io) for in-depth user documentation.
+
+Additional docs can be found in the `/docs` directory, and the [index is here](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/docs).
+
+## Getting involved and contributing
+
+Are you interested in contributing to cluster-api-provider-ibmcloud? We, the
+maintainers and community, would love your suggestions, contributions, and help!
+Also, the maintainers can be contacted at any time to learn more about how to get
+involved.
+
+In the interest of getting more new people involved, we tag issues with
+[`good first issue`](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues?q=is%3Aissue+label%3A%22good+first+issue%22+is%3Aopen).
+These are typically issues that have smaller scope but are good ways to start
+to get acquainted with the codebase.
+
+We also encourage all active community participants to act as if they are
+maintainers, even if you don't have "official" write permissions. This is a
+community effort, we are here to serve the Kubernetes community. If you have an
+active interest and you want to get involved, you have real power! Don't assume
+that the only people who can get things done around here are the "maintainers".
+
+We also would love to add more "official" maintainers, so show us what you can
+do!
+
+This repository uses the Kubernetes bots.  See a full list of the commands [here](https://prow.k8s.io/command-help).
+
+### Join us
+
+The community holds a weekly meeting every Friday at 09:00 (IST) / 03:30 (UTC) on [Zoom](https://zoom.us/j/9392903494). Subscribe to the [SIG Cluster Lifecycle](https://groups.google.com/g/kubernetes-sig-cluster-lifecycle) Google Group for access to documents and calendars
+
+
+### Other ways to communicate with the contributors
+
+Please check in with us in the [#cluster-api-ibmcloud](https://kubernetes.slack.com/archives/C02F4CX3ALF) channel on Slack.
+
+## Github issues
+
+### Bugs
+
+If you think you have found a bug please follow the instructions below.
+
+- Please spend a small amount of time giving due diligence to the issue tracker. Your issue might be a duplicate.
+- Get the logs from the cluster controllers. Please paste this into your issue.
+- Open a [bug report](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/new?assignees=&labels=&template=bug_report.md).
+- Remember users might be searching for your issue in the future, so please give it a meaningful title to helps others.
+- Feel free to reach out to the cluster-api community on [kubernetes slack](https://kubernetes.slack.com/archives/C8TSNPY4T).
```

Comment:
any reason for mentioning the `cluster-api` slack channel instead of `cluster-api-ibmcloud`?

Reply:
@mkumatag  , I already the `cluster-api-ibmcloud` slack channel under a different section on L62. To avoid confusion, I have removed this.